exampleSite/content/index/awesome-team/yoda.md

```diff
@@ -0,0 +1,37 @@
++++
+  title = "Tiny Gopher 3"
+  weight = 30
+  joined = "2017-10-17"
+  image = "tinygopher.png"
+  position = "Gopherineer"
+  
+  reportsTo = "Lead Gophineer"
+  livesIn = "[Iceland](https://www.google.com/maps/place/Iceland/)"
+  scope = [
+    "Gopherineering for various tasks",
+    "Contributor for [Goper Team A](#)",
+    "Gopher Community Member"
+  ]
+
+
+  [[member.icons]]
+    icon = "fa-twitter"
+    link = "#"
+  [[member.icons]]
+    icon = "fa-facebook-f"
+    link = "#"
+  [[member.icons]]
+    icon = "fa-linkedin-in"
+    link = "#"
++++
+    
+  description = """
```

Comment:
Still the `description = """` part is not necessary at all. It will print on the page.

Reply:
I'll fix this.